README.md

```diff
@@ -158,6 +158,75 @@ elm-i18n-generator --format PO --language De --import export.po
 Results in the same `import/De/Translation/Main.elm`
 as in the [CSV example](#import-generate-elm-source-code-from-csv).
 
+
+#### Generate Elm source for switching Language during run-time
+
+Pass all the languages you want to switch between as a comma-separated list.
+
+```bash
+elm-i18n-generator --language De,Pl,En,Fr --genswitch
+```
+
+#### Migrate legacy symlinked translations to switchables
```

Comment:
We should maybe not just yet call it legacy. I believe that both options could reside side-by-side with advantages for each approach.

Reply:
Ideally we need to implement this in a way that does not break existing implementation unless they use new cli flags.